app/views/site/_view_gene_expression_title_bar.html.erb

```diff
@@ -8,14 +8,13 @@
 </div>
 
 <script>
-  // Initializes Ideogram for related genes
 
-  document.querySelector('.study-lead').parentElement.innerHTML +=
-    '<div id="related-genes-ideogram-container" class="hidden-related-genes-ideogram"></div>';
-
-  // TODO (SCP-2769): Handle when a searched gene maps to multiple species
-  window.SCP.taxon = '<%= @taxons.first %>';
-
-  createRelatedGenesIdeogram();
+  // Initializes Ideogram for related genes, if it hasn't already been
+  // started for the current searched gene.
+  if (window.SCP.taxons.length > 1) {
```

Comment:
I'm not sure I understand the flow between this and the above setting of window.SCP.taxons.  Why aren't window.SCP.taxon and window.SCP.taxons set at the same time?  And is window.SCP.taxon just the first taxon in the window.SCP.taxons array?  That seems like a needless variable

Reply:
I also don't understand why we would only create the ideogram container if there are more than one taxon

Reply:
ok, now I think I've figured it out.  oof, that is some tricky state-passing between rails templates and JS.  I'm trying to think of ways to clean it -- maybe have createRelatedGenesIdeogram() take a parameter of the taxon to render?   that saves a global variable

Reply:
Tricky indeed.  Now mitigated per your suggestion in 0ba699c.